code/modules/vending/vendor/interaction.dm

```diff
@@ -1,3 +1,20 @@
+//===============================HAND INTERACTION===================================
+/obj/machinery/vending/interact(mob/user)
+	if (HAS_AI_ACCESS(user))
```

Comment:
Should do Adjacent instead otherwise a human AI with the wand out can't right the vendor

Reply:
Put your change as a suggestion so I can commit it